docs/pages/installation/gke.md

```diff
@@ -5,19 +5,32 @@ permalink: installation/gke
 ---
 
 # Quick Start with Google Kubernetes Engine (GKE)
-You may perform the steps outlined under [Managed Kubernetes](#managedk8s) following by hand or run the [generate_kubeconfig_gke.sh](./generate_kubeconfig_gke.sh) shell script using the desired ServiceAccount name and Namespace arguments, like so:
 
-`./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
+- Navigate to the Meshery UI, login with your user details and head over to the local port (e.g. `http://localhost:9081`).
+- Download the authentication token by clicking the "Get Token" option in the dropdown menu under your User Account avatar.
+- Utilize the token to run the following command:
+
+| command           | flag                | function                                                     | Usage                     |
+|:------------------|:-------------------:|:-------------------------------------------------------------|:--------------------------|
+|                   | --system config     | configures Meshery with the kubeconfig, generated with the help of user details, to provide cluster access for public clouds(GKE). | `mesheryctl system config gke --token "PATH TO TOKEN"` |
+
+Once configured, you can proceed to the Quick Start Guide and continue with the steps outlined for [GKE](/docs/installation/gke#manual-configuration).
```

Comment:
A link to the Quick Start guide is needed, not a link back to this same document.